c/src/answer.rs

```diff
@@ -43,7 +43,7 @@ pub extern "C" fn concept_map_drop(concept_map: *mut ConceptMap) {
 
 #[no_mangle]
 pub extern "C" fn concept_map_get_variables(concept_map: *const ConceptMap) -> *mut StringIterator {
-    release(StringIterator(CIterator(box_stream(borrow(concept_map).map.clone().into_keys()))))
+    release(StringIterator(CIterator(box_stream(borrow(concept_map).map.clone().into_keys().map(Ok)))))
```

Comment:
`StringIterator` is fallible now that `query_fetch()` returns a stream of JSONs.